radix-engine-common/src/data/scrypto/custom_schema.rs

```diff
@@ -30,16 +33,10 @@ pub enum ScryptoCustomTypeKind {
 
 impl<L: SchemaTypeLink> CustomTypeKind<L> for ScryptoCustomTypeKind {
     type CustomValueKind = ScryptoCustomValueKind;
-    type CustomTypeExtension = ScryptoCustomTypeExtension;
 }
 
-#[derive(Debug, Clone, PartialEq, Eq, Sbor)]
-pub enum ScryptoCustomTypeValidation {}
-
-impl CustomTypeValidation for ScryptoCustomTypeValidation {}
-
 #[derive(Debug, Clone, PartialEq, Eq, Copy)]
-pub enum ScryptoCustomTypeExtension {}
+pub struct ScryptoCustomTypeExtension {}
```

Comment:
Like you said, there isn't much difference since it's empy. 

It just happened during the iteration; at one point, I wanted to make `ScryptoCustomTypeExtension` stateful.